libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackClientWrapper.cs

```diff
@@ -94,8 +95,9 @@ public virtual async Task<LoginResponse> ConnectAsync()
         /// </summary>
         /// <param name="channelId">The channel to delete the message from.</param>
         /// <param name="ts">The timestamp of the message.</param>
+        /// <param name="cancellationToken">A cancellation token for the task.</param>
         /// <returns>A <see cref="DeletedResponse"/> representing the response to deleting the message.</returns>
-        public virtual async Task<DeletedResponse> DeleteMessageAsync(string channelId, DateTime ts)
+        public virtual async Task<DeletedResponse> DeleteMessageAsync(string channelId, DateTime ts, CancellationToken cancellationToken)
```

Comment:
Shouldn't the cancellationToken be `CancellationToken cancellationToken = default(CancellationToken)`?